scripts/nb-tester/qiskit_docs_notebook_tester/__init__.py

```diff
@@ -102,6 +117,20 @@ def notebooks_to_execute(self) -> Iterator[Path]:
 
             yield path
 
+    def check_all_notebooks_are_classified(self) -> None:
+        unclassified = [
+            path for path in Path(".").glob("[!.]*/**/*.ipynb")
+            if not matches(path, self.all_notebooks)
+        ]
+        if unclassified == []:
+            return
+        print(
+            f"\nThe following notebooks are not classified in {self.args.config_path}:\n  "
+            + "\n  ".join(map(str, unclassified))
+            + "\nAdd them to the appropriate group so we know how to test them.\n"
+        )
+        sys.exit(1)
```

Comment:
You could do this instead. https://adamj.eu/tech/2021/10/10/the-many-ways-to-exit-in-python/. It doesn't really matter, so feel free to disagree. Only practical difference here is writing to stderr rather than stdout.

```suggestion
        raise SystemExit(
            f"\nThe following notebooks are not classified in {self.args.config_path}:\n  "
            + "\n  ".join(map(str, unclassified))
            + "\nAdd them to the appropriate group so we know how to test them.\n"
        )
```

Reply:
Nice, thanks!